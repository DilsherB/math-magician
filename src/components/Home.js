import React from "react";

const Home = () => {
  return (
    <div>
      <h1>Hi there!</h1>
      <p className="text-left bg-dark">
        Welcome to math Magicians website.
        <br />
        Here you can do some basic calculations by going to Calculator tab.
        <br />
        You can also read some beatiful quotes by going to Quote tab.
        <br />
        This web app is build by me(Dilsher Balouch). I used following techs
        here:
        <br />
        <ol>
          <li>React</li>
          <li>React Router</li>
          <li>React Hooks</li>
          <li>React Bootstrap</li>
          <li>ESLint</li>
          <li>Netlify</li>
        </ol>{" "}
      </p>
    </div>
  );
};

export default Home;
