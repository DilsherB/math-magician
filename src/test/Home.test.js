import renderer from "react-test-renderer";
import React from "react";
import Home from "../components/Home";

const renderTree = (tree) => renderer.create(tree);
describe("<Home>", () => {
  it("should render component", () => {
    expect(renderTree(<Home />).toJSON()).toMatchSnapshot();
  });
});
