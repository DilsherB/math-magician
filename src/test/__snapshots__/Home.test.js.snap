// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home> should render component 1`] = `
<div>
  <h1
    className="text-center"
  >
    Hi there!
  </h1>
  <p
    className="w-50"
    style={
      Object {
        "margin": "0 auto",
      }
    }
  >
    Welcome to math Magicians website.
    <br />
    This web app is build by me(Dilsher Balouch).
    <ul>
      <li>
        Here you can do some basic calculations by going to Calculator tab.
      </li>
      <li>
        You can also read some beatiful quotes by going to Quote tab.
      </li>
    </ul>
    <br />
     I used following techs here:
    <ol>
      <li>
        React
      </li>
      <li>
        React Router
      </li>
      <li>
        React Hooks
      </li>
      <li>
        React Bootstrap
      </li>
      <li>
        ESLint
      </li>
      <li>
        Netlify
      </li>
    </ol>
  </p>
</div>
`;
